hooks/environment

```diff
@@ -66,12 +66,14 @@ else
   check_command jq
   secret=$(retry vault kv get -format=json "${BUILDKITE_PLUGIN_VAULT_SECRETS_PATH}" | jq -c .data)
 fi
-
+echo "${secret}" | buildkite-agent redactor add
 if [ -n "${BUILDKITE_PLUGIN_VAULT_SECRETS_ENV_VAR}" ]; then
   eval "${BUILDKITE_PLUGIN_VAULT_SECRETS_ENV_VAR}='${secret}'"
   export "${BUILDKITE_PLUGIN_VAULT_SECRETS_ENV_VAR?}"
+  echo "${BUILDKITE_PLUGIN_VAULT_SECRETS_ENV_VAR}" | buildkite-agent redactor add
```

Comment:
Shouldn't be needed, as this is a repeat of echoing `secret`
```suggestion
```

Reply:
Lets take the discussion under the next thread.